legal-api/src/legal_api/reports/report.py

```diff
@@ -176,10 +176,11 @@ def _set_dates(self, filing):
         filing['effective_date_time'] = effective_date.strftime(f'%B %-d, %Y at {effective_hour}:%M %p Pacific Time')
         filing['effective_date'] = effective_date.strftime('%B %d, %Y')
         # Recognition Date
-        recognition_datetime = LegislationDatetime.as_legislation_timezone(self._business.founding_date)
-        recognition_hour = recognition_datetime.strftime('%I').lstrip('0')
-        filing['recognition_date_time'] = \
-            recognition_datetime.strftime(f'%B %-d, %Y at {recognition_hour}:%M %p Pacific Time')
+        if self._business:
```

Comment:
This is just to support testing right?

Reply:
No, this is the fix for the issue you mentioned yesterday.. unable to download IA report for a paid filing

Reply:
How is it that the other code in this file that de-references self._business doesn't bomb?
Eg, lines 137, 144 and 275

Reply:
Please see the condition above the specified line numbers

Reply:
OK. Resolving this conversation.